zerver/openapi/zulip.yaml

```diff
@@ -156,8 +156,39 @@ paths:
                           guaranteed to be increasing, but they are not guaranteed to be
                           consecutive.
                         items:
-                          type: object
-                          additionalProperties: false
+                          oneOf:
+                          - type: object
+                            properties:
+                              type:
+                                description: |
+                                  Alert words
+                                type: string
+                                enum:
+                                - alert_words
+                              alert_words:
+                                type: array
+                                items:
+                                 type: string
+                            additionalProperties: false
```

Comment:
Can we provide examples in this scope?  I feel like they'd be really helpful.

Reply:
Yes I verified it.